composables/useCart.ts

```diff
@@ -93,6 +95,26 @@ export const useCart = () => {
     }
   }
 
+  const updateCartItem = async (cartItemId: string, cartItemInput: CartItemInput) => {
+    const variables = {
+      cartItemId,
+      cartItemInput,
+    }
+    try {
+      loading.value = true
+      await fetcher({
+        query: updateCartItemMutation,
+        variables,
+      })
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err)
+    } finally {
+      loading.value = false
+      cart.value = await getCart()
```

Comment:
I think cart.value = await getCart() should be in Try block

Reply:
Done